### PR TITLE
infrequent tier monitors and ARP

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]

File: docs/manage/partitions-data-tiers/infrequent-tier-monitors.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Previously, monitors and alert response page (ARP) were only supported in Sumo Logic’s Continuous data tier. Now, you can set robust and configurable alerting policies that enable you to get notified about critical changes and view granular details about an individual alert that include data from the Infrequent tier.
+
+## Monitors
+
+Monitors track your Metrics, Logs, or SLO infrequent data in real time and send notifications when noteworthy changes happen in your production applications. To create Monitors, you can refer to the [Create a Monitor](/docs/alerts/monitors/create-monitor/).
+
+## Alert Response Page
+
+The Alert page is where you can view granular details about an individual alert. [Learn more](/docs/alerts/monitors/alert-response/#alert-page).
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/infrequent-tier-monitors.md:21]:
```suggestion
The Alert Response page is where you can view granular details about an individual alert. [Learn more](/docs/alerts/monitors/alert-response/#alert-page).
```

File: docs/manage/partitions-data-tiers/infrequent-tier-monitors.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+id: infrequent-tier-monitors
+title: Monitors and Alert Response Page on Infrequent Tier data (Beta)
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/infrequent-tier-monitors.md:3]:
```suggestion
title: Monitors and Alert Response Page on Infrequent Tier Data (Beta)
```

File: docs/manage/partitions-data-tiers/infrequent-tier-monitors.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Previously, monitors and alert response page (ARP) were only supported in Sumo Logic’s Continuous data tier. Now, you can set robust and configurable alerting policies that enable you to get notified about critical changes and view granular details about an individual alert that include data from the Infrequent tier.
+
+## Monitors
+
+Monitors track your Metrics, Logs, or SLO infrequent data in real time and send notifications when noteworthy changes happen in your production applications. To create Monitors, you can refer to the [Create a Monitor](/docs/alerts/monitors/create-monitor/).
+
+## Alert Response Page
+
+The Alert Response page is where you can view granular details about an individual alert. [Learn more](/docs/alerts/monitors/alert-response/#alert-page).
```
Comment (kimsauce) [inline docs/manage/partitions-data-tiers/infrequent-tier-monitors.md:21]:
```suggestion
The Alert page is where you can view granular details about an individual alert. [Learn more](/docs/alerts/monitors/alert-response/#alert-page).
```

**Review by kimsauce — APPROVED** [review]

File: docs/manage/partitions-data-tiers/infrequent-tier-monitors.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Closed Beta</span></a></p>
+
+Previously, monitors and alert response page (ARP) were only supported in Sumo Logic’s Continuous data tier. Now, you can set robust and configurable alerting policies that enable you to get notified about critical changes and view granular details about an individual alert that include data from the Infrequent tier.
+
+## Monitors
+
+Monitors track your Metrics, Logs, or SLO infrequent data in real time and send notifications when noteworthy changes happen in your production applications. To create Monitors, you can refer to the [Create a Monitor](/docs/alerts/monitors/create-monitor/).
```
Comment (tarunk2) [inline docs/manage/partitions-data-tiers/infrequent-tier-monitors.md:17]: First line seems to indicate that monitors track infrequent data only. We can remove `infrequent` keyword from there.

Comment (JV0812) [reply docs/manage/partitions-data-tiers/infrequent-tier-monitors.md]: Thanks @tarunk2. I have incorporated your comment.

**Pull request merged**